src/modal/modal-window.ts

```diff
@@ -21,7 +22,8 @@ import {ModalDismissReasons} from './modal-dismiss-reasons';
     'role': 'dialog',
     'tabindex': '-1',
     '(keyup.esc)': 'escKey($event)',
-    '(click)': 'backdropClick($event)'
+    '(click)': 'backdropClick($event)',
+    '[attr.aria-labelledby]': 'isDefined(ariaLabelledBy) ? ariaLabelledBy : null',
```

Comment:
looks like an overkill for me, as attribute won't be there for `undefined` and `null` values anyway

Reply:
true, that's something I used to be unaware of, I'll remove the check